### PR TITLE
IGNITE-22897 CLI for initiating MG repair

### PR DESCRIPTION
**Pull request by Phillippko**
https://issues.apache.org/jira/browse/IGNITE-22897

### CODE REVIEW

File: modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/Options.java [hunk 1]
```diff
@@ -292,6 +292,11 @@ public static final class Constants {
         public static final String RECOVERY_NODE_NAMES_OPTION_DESC = "Names specifying nodes to get partition states from. "
                 + "Case-sensitive, without quotes, all nodes if not set";
 
+        public static final String RECOVERY_METASTORAGE_REPLICATION_OPTION = "--metastorage-replication-factor";
+
+        public static final String RECOVERY_METASTORAGE_REPLICATION_DESC = "Number of nodes in the voting member set of the Metastorage "
```
Comment (Phillippko) [inline modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/Options.java:297]: (Will consult with Alex about args descriptions)

File: modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/recovery/cluster/reset/ResetClusterMixin.java [hunk 2]
```diff
@@ -19,17 +19,34 @@
...
+    public @Nullable List<String> cmgNodeNames() {
+        return options.cmgNodeNames;
+    }
+
+    /** Returns metastorage replication factor. */
+    public @Nullable Integer metastorageReplicationFactor() {
+        return options.metastorageReplicationFactor;
+    }
+
+    private static class Options {
```
Comment (rpuch) [inline modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/recovery/cluster/reset/ResetClusterMixin.java:45]: What's the purpose of this class? Why couldn't we just declare 2 fields in the mixin itself?

**Review by rpuch — COMMENTED** [review]
There are still mentions of IGNITE-22897 in code

File: modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/recovery/cluster/reset/ResetClusterMixin.java (hunk 2 above)
Comment (Phillippko) [reply modules/cli/src/main/java/org/apache/ignite/internal/cli/commands/recovery/cluster/reset/ResetClusterMixin.java:45]: To use picocli's ArgGroup and check that at least one of this fields is specified

**Review by rpuch — APPROVED** [review]

**Pull request merged**